packages/driver/cypress/e2e/cypress/element_selector.cy.ts

```diff
@@ -0,0 +1,86 @@
+/// <reference types="cypress" />
```

Comment:
Converted to TypeScript - the tests are the same except for removing tests around `onElement`